hypha/apply/users/tests/factories.py

```diff
@@ -67,6 +68,32 @@ def groups(self, create, extracted, **kwargs):
             self.groups.add(GroupFactory(name=STAFF_GROUP_NAME))
 
 
+def get_wagtail_admin_access_permission():
```

Comment:
Ideally this would be a fully generic `get_permission_from_perm_string` similar to what is described in [this StackOverflow question](https://stackoverflow.com/a/40892015/159522) but I wasn't sure where to put that kind of code.